README.md

```diff
@@ -67,7 +70,7 @@ After writing this method, our new result should be:
 ...
 ```
 
-**think**: Where should we call `strip_punctuation`? Does it matter? Why?
+**think**: Where should we call `strip_punctuation`? Does it matter? Why? You can call strip_punctuation right after line 14 because this is the only point in the code where the words are being analyzed individually or replace element in line 38. I think it's easier to read if you put it in line 14.
```

Comment:
This is a great answer! I was also looking for the fact that it belong in the analyze_mood method because there shouldn't be a situation where we want to analyze_mood without strip_punctuation.